docs/shared-content-source/docs/modules/develop/pages/cloudflow-configuration.adoc

```diff
@@ -149,6 +149,52 @@ The pod is limited to use `4Gi` of memory.
 It also shows how to set options to the JVM through an environment variable
 and how to add labels to a pod. 
 
+It also possible to add `volumes`, that contain secrets, to the pods. And then mount this volumes into the container with `volume-mounts`. Bear in mind that each `volume-mounts` name has to match a `volumes`  name, defined in the same config file. Here's a sample of how to set two `volumes` with two different secrets and how to mount them in a container.  
+
+[source, hocon]
+----
+cloudflow.streamlets.my-streamlet {
+  kubernetes.pods.pod {
+   volumes {
+      foo {
+        secret {
+          name = mysecret
+        }
+      }
+      bar {
+        secret {
+          name = mysecret2
+        }
+      }
+    }
+    containers.container {
+      volume-mounts {
+        foo {
+          mountPath = "/mnt/folderA"
+          readOnly = true
+        }
+        bar {
+          mountPath = "/mnt/folderB"
+          readOnly = true
+        }
+      }
+    }
+  } 
+}
+----
+
+This configuration will mount `mysecret` into each `my-streamlet` container on to the path `/mnt/folderA` as read only, while `mysecret2` will be mounted on path `/mnt/folderB`. Each `volume-mounts` name is matching a `volumes` name as `volume-mounts.foo` matches `volumes.foo` and `volume-mounts.bar` matches `volumes.bar`
```

Comment:
```suggestion
This configuration will mount `mysecret` into each `my-streamlet` container on to the path `/mnt/folderA` as read only, while `mysecret2` will be mounted on path `/mnt/folderB`. Each `volume-mounts` name is matching a `volumes` name as `volume-mounts.foo` matches `volumes.foo` and `volume-mounts.bar` matches `volumes.bar`.
```

Reply:
is `Each `volume-mounts` name must matching a `volumes` name` right @RayRoestenburg shouldn't it be `Each `volume-mounts` name must match a `volumes` name`